### PR TITLE
Improve tutorial scenarios

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose

Add MFA basic tutorial

Adjust scenarios and intros of the following tutorials:

- Adding users and roles
- Configuring Single Sign On Using SAML
- Configuring Single Sign On using OIDC

### CODE REVIEW

File: en/docs/learn/adding-users-and-roles.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-## Scenario
-This tutorial demonstrates user and role management in WSO2 Identity Server. User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
+User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
 
 This tutorial demonstrates how to create a new user, create a role with a set of permissions, and then log in to the dashboard as the newly created user. 
 
+## Scenario
+
+Pickup is a cab company that has many employees. Each of these employees have a particular designation such as "Manager", 
+"Driver" etc., and depending on their designation within the company, they will have different permissions and access levels.
```
Comment (YvonneW) [inline en/docs/learn/adding-users-and-roles.md]: The sentence is split into two lines.

Comment (sherenewso2) [reply en/docs/learn/adding-users-and-roles.md]: Fixed with aef9696f95c1cfc17cb8624c017f1f43833c82c8

File: en/docs/learn/adding-users-and-roles.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
 # Adding Users and Roles
 
-## Scenario
-This tutorial demonstrates user and role management in WSO2 Identity Server. User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
+User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
 
 This tutorial demonstrates how to create a new user, create a role with a set of permissions, and then log in to the dashboard as the newly created user.
```
Comment (YvonneW) [inline en/docs/learn/adding-users-and-roles.md]: WSO2 Identity Server Dashboard with the newly created user's credentials.

File: en/docs/learn/configuring-multifactor-authentication.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+    ``` java tab="Linux/MacOS"
+    sh wso2server.sh
+    ```
+
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+
+3. Access the [management console]() and login using admin/admin credentials.
```
Comment (gomathyk) [inline en/docs/learn/configuring-multifactor-authentication.md]: The link for management console is missing.

File: en/docs/learn/configuring-multifactor-authentication.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        SMSOTPMandatory = false
+        CaptureAndUpdateMobileNumber = true
+        SendOTPDirectlyToMobile = false
+        redirectToMultiOptionPageOnFailure = false
+        ```
+
+    !!! info
+        For more information about these configurations, see [Configuring SMS OTP](../../learn/configuring-sms-otp).
+
+2. Download the certificate of the SMS provider by going to the SMS providers website on your browser, and clicking the HTTPS trust icon on the address bar.
```
Comment (gomathyk) [inline en/docs/learn/configuring-multifactor-authentication.md]: provider's

Comment (sherenewso2) [reply en/docs/learn/configuring-multifactor-authentication.md]: Fixed 
485f165

File: en/docs/learn/configuring-multifactor-authentication.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-multifactor-authentication.md]: Fixed 
485f165

File: en/docs/learn/adding-users-and-roles.md (hunk 2 above)
Comment (sherenewso2) [reply en/docs/learn/adding-users-and-roles.md]: Fixed 
485f165

**Pull request merged**